src/amount.cpp

```diff
@@ -12,14 +12,14 @@ const std::string CURRENCY_UNIT = "BTC";
 CFeeRate::CFeeRate(const CAmount& nFeePaid, size_t nSize)
 {
     if (nSize > 0)
-        nSatoshisPerK = nFeePaid*1000/nSize;
+        nSatoshisPerK = nFeePaid * PRECISION_MULTIPLIER / nSize;
```

Comment:
You can't call this `nSatoshisPerK` anymore. If you break something, change its name.